src/lib/classes/lsp3-universal-profile.spec.ts

```diff
@@ -65,4 +65,32 @@ describe('LSP3UniversalProfile', () => {
       },
     });
   });
+  it.skip('should deploy and set LSP3Profile data', async () => {
```

Comment:
`.skip` is intended?

Reply:
it is yes. This test is WIP as there was some problems with the Jest config causing type errors we couldn't get to the bottom of yet. Merged so we can continue with release